src/main/tables/95_async_tasks.sql

```diff
@@ -0,0 +1,14 @@
+SET search_path = public, pg_catalog;
+
+--
+-- async_tasks table
+--
+CREATE TABLE async_tasks (
+    id uuid NOT NULL DEFAULT uuid_generate_v1(),
+    type text NOT NULL,
```

Comment:
I'm guessing `type` will be something like `data`, `analysis`, etc.?

Reply:
More specific than that was my plan -- it'd be stuff like "file move" "metadata save" "url import" etc.

basically if a service that processes async tasks wants to query the `async-tasks` service and find everything that it knows how to process, it'll use this type as a filter. I could have it be by service, but I'd rather have the granularity (without having to look into the `data` field) to drill down to very specific types of tasks. I'd considered adding a type _and_ a subtype field, and if I did that it'd be like you said for this field, with the more specific info in the subtype. But it seemed like overkill to start, unless you feel strongly about it :)

Reply:
... heh, I'm realizing I should probably add an index on that field, and probably on username and the date fields too. Maybe I'll do that now, in this PR.

Reply:
OK, nice, that makes sense. I have no strong preference for now. Though now that you mention it, it could be convenient for some service to query for all `filesystem` events, then it could loop through them and switch on the subtype (`move`, `delete`, `rename`, etc.), rather than having to query for all `filesystem-move` events, then query for all `filesystem-delete` events separately 🤔

Reply:
Oh, heh, and I was just figuring the filter parameter would allow it to provide a list. Which I suppose is basically equivalent. One way or the other the service has to know what it's capable of processing, I guess